tools/shoestring/shoestring/commands/import_bootstrap.py

```diff
@@ -32,4 +41,5 @@ async def run_main(args):
 def add_arguments(parser):
 	parser.add_argument('--config', help=_('argument-help-config'), required=True)
 	parser.add_argument('--bootstrap', help=_('argument-help-import-bootstrap-bootstrap'), required=True)
+	parser.add_argument('--include-node-key', help=_('argument-help-import-bootstrap-node-key'), action='store_true')
```

Comment:
minor - consider updating resource name to:  `argument-help-import-bootstrap-include-node-key`